test/form/samples/emit-uint8array-no-buffer/_expected/umd.js

```diff
@@ -3,15 +3,15 @@
 	factory();
 }((function () { 'use strict';
 
-	var asset1a = (typeof document === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-dc5cb674').href : new URL('assets/asset-dc5cb674', document.currentScript && document.currentScript.src || document.baseURI).href);
+	var asset1a = (typeof document === 'undefined' ? (typeof location !== 'undefined' ? location.href : new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-dc5cb674').href) : new URL('assets/asset-dc5cb674', document.currentScript && document.currentScript.src || document.baseURI).href);
 
-	var asset1b = (typeof document === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-dc5cb674').href : new URL('assets/asset-dc5cb674', document.currentScript && document.currentScript.src || document.baseURI).href);
+	var asset1b = (typeof document === 'undefined' ? (typeof location !== 'undefined' ? location.href : new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-dc5cb674').href) : new URL('assets/asset-dc5cb674', document.currentScript && document.currentScript.src || document.baseURI).href);
 
-	var asset2a = (typeof document === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-52cbd095').href : new URL('assets/asset-52cbd095', document.currentScript && document.currentScript.src || document.baseURI).href);
+	var asset2a = (typeof document === 'undefined' ? (typeof location !== 'undefined' ? location.href : new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-52cbd095').href) : new URL('assets/asset-52cbd095', document.currentScript && document.currentScript.src || document.baseURI).href);
 
-	var asset2b = (typeof document === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-52cbd095').href : new URL('assets/asset-52cbd095', document.currentScript && document.currentScript.src || document.baseURI).href);
+	var asset2b = (typeof document === 'undefined' ? (typeof location !== 'undefined' ? location.href : new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-52cbd095').href) : new URL('assets/asset-52cbd095', document.currentScript && document.currentScript.src || document.baseURI).href);
 
-	var asset99a = (typeof document === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-c568a840').href : new URL('assets/asset-c568a840', document.currentScript && document.currentScript.src || document.baseURI).href);
+	var asset99a = (typeof document === 'undefined' ? (typeof location !== 'undefined' ? location.href : new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-c568a840').href) : new URL('assets/asset-c568a840', document.currentScript && document.currentScript.src || document.baseURI).href);
```

Comment:
Apologies I didn't review this fully - we need to ensure we are doing the correct normalization here:

```suggestion
	var asset99a = (typeof document === 'undefined' && typeof location === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __dirname + '/assets/asset-c568a840').href) : new URL('assets/asset-c568a840', typeof document !== 'undefined' ? document.currentScript && document.currentScript.src || document.baseURI : location.href).href);
```

Reply:
Fixed